app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -3870,4 +3870,62 @@ class PatientRepositoryAndroidTest {
       assertThat(syncStatus).isEqualTo(PENDING)
     }
   }
+
+  @Test
+  fun `searching_for_a_patient_by_phone_number_must_return_list_of_patient_search_results_paritioned_by_assigned_and_not_assigned_facility`() {
+    fun createPatientWithPhoneNumberAndAssignedFacilityID(
+        patientUuid: UUID,
+        phoneNumber: String?,
+        assignedFacilityId: UUID?
+    ) {
+      val patientProfile = testData
+          .patientProfile(patientUuid = patientUuid, generatePhoneNumber = !phoneNumber.isNullOrBlank(), patientAssignedFacilityId = assignedFacilityId)
+          .let { patientProfile ->
+            patientProfile.copy(
+                patient = patientProfile.patient.copy(fullName = "Patient", assignedFacilityId = assignedFacilityId),
+                phoneNumbers = patientProfile
+                    .phoneNumbers
+                    .take(1)
+                    .map { it.copy(number = phoneNumber!!) }
+            )
+          }
+
+      patientRepository.save(listOf(patientProfile)).blockingAwait()
+    }
+
+    fun searchResults(phoneNumber: String, facilityUuid: UUID): Set<UUID> {
+      return patientRepository
+          .search2(PhoneNumber(phoneNumber), facilityUuid)
+          .map { it.uuid }
+          .toSet()
```

Comment:
Is there a reason we are converting them to `Set` here?

Reply:
Not here as I am only creating one search result object

Reply:
yeah, but wouldn't a list suffice for this?

Reply:
I have removed it from here as I'm only using this method once so I inlined it.